ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/ptp.j2

```diff
@@ -31,7 +31,11 @@ ptp message-type general dscp {{ ptp.message_type.general.dscp }} default
 ptp message-type event dscp {{ ptp.message_type.event.dscp }} default
 {%     endif %}
 {%     if ptp.mode is arista.avd.defined %}
+{%         if (ptp.mode is arista.avd.defined('boundary') or ptp.mode is arista.avd.defined('e2etransparent')) and ptp.mode_one_step is arista.avd.defined(true) %}
```

Comment:
technically we don't support `e2etransparent` mode today so maybe let's just add a comment to add this later once we implement it in the valid_values

Reply:
Fixed